File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar() {
        echo "
            Licencia: $this->license 
            Driver: {$this->driver->name} 
            Marca: $this->marca
            Modelo: $this->model
            Número de pasajeros: $this->passenger
    
        ";
    }
}
?>

File: Python/UberX.py
from lib2to3.refactor import _Driver
from pyexpat import model
from car import Car

class UberX(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super.__init__(license,driver)
        self.brand = brand
        self.model = model